pytest_cookies.py

```diff
@@ -14,17 +17,26 @@
 class Result(object):
     """Holds the captured result of the cookiecutter project generation."""
 
-    def __init__(self, exception=None, exit_code=0, project_dir=None):
+    def __init__(self, exception=None, exit_code=0, project_dir=None,
+                 trace_back=None):
         self.exception = exception
         self.exit_code = exit_code
         self._project_dir = project_dir
+        self.trace_back = trace_back
 
     @property
     def project(self):
         if self.exception is None:
             return py.path.local(self._project_dir)
         return None
 
+    @property
+    def trace_back_stack(self):
```

Comment:
I'm not convinced that we need this- consumers of `pytest-cookies` can transform the traceback themselves?

Reply:
Since it was trivial to add, it follows that it is trivial for the users of pytest-cookies to do on their side of the interface as well. I am okay with not providing it.

Reply:
👍